reana-ui/src/pages/launchOnReana/LaunchOnReana.module.scss

```diff
@@ -58,4 +69,23 @@
     font-size: 0.8em;
     width: 100%;
   }
+
+  .code-snippet-table-cell {
+    padding: 0 !important;
+
+    div {
+      margin: 0;
+    }
+  }
+
+  .loader {
+    padding: 20px;
+    background-color: white;
+    border-radius: 5px;
+
+    &:before,
+    &:after {
+      top: 10px !important;
+    }
```

Comment:
Let's keep the default loader distance? Or is there a particular reason to use a custom distance only for the launcher?